test/code/plugins/oms_common_test.rb

```diff
@@ -94,6 +100,256 @@ def teardown
       "OSAlias=UniversalR\n" \
       "OSManufacturer=Central Logistics GmbH\n"
 
+    # Begin:  Unit Tests for some simple methods.
+    # May best be put in another file later.
+
+    def test_clean_hostname_string
+      hn = ' www.five.com:333     '
+      b = Common.clean_hostname_string(hn)
```

Comment:
You can eliminate at least 4 lines in this method; you should not declare your test string as a meaningless variable rather than just directly passing it to `clean_hostname_string()`. In this case, repeated code will make people less likely to check that your test cases are correct and thorough.

Reply:
The advantage of buffer variables is that trace prints can be embedded quickly.  In these tests extra lines do not hurt runs.  One can also re-use them in some tests for comparisons.  However, readability is certainly of concern.  I'll go through it with that in mind.  Thanks.

Reply:
I took some of these out and made better names to some.